scheduler/src/cook/config.clj

```diff
@@ -335,11 +343,13 @@
      :mesos-framework-name (fnk [[:config {mesos nil}]]
                              (when mesos
                                (or (:framework-name mesos) "Cook")))
-     :mesos-framework-id (fnk [[:config {mesos nil}]]
-                           (:framework-id mesos))
-     :mesos-compute-cluster-name (fnk [[:config {mesos nil}]]
-                                   (or (:compute-cluster-name mesos) "default-compute-cluster-from-config-defaulting"))
-
+     ; TODO(pschorf): Remove
```

Comment:
Is this TODO for this PR or a future one?

Reply:
Future PR (currently WIP, but still want to merge separately)